tools/README.md

```diff
@@ -12,7 +12,8 @@ GPU generated event logs.
 ## Prerequisites
 - Spark 3.0.1 or newer installed
 - Java 8 or above
-- Complete Spark event log(s) from Spark 3.0 or above version.
+- Complete Spark event log(s) from Spark 3.0 or above version
```

Comment:
so one other thing we might mention is how to find event logs.  Perhaps reference spark config 

spark.eventLog.dir



<br class="Apple-interchange-newline">

Reply:
I would suggest they just use UI , and there is a download button on the right side.
It is easier, especially for CSP users because many CSP users may not even ssh to the nodes.

Reply:
Just added on how to download spark event log.

Reply:
can we say both, because if the applications have already run, they can look in the event log directory. You can find it by look at the spark config spark.eventLog.dir

Reply:
if you push another change ask it to build so the build goes overnight

Reply:
Sure. Done.